packages/react-server-dom-webpack/src/ReactFlightWebpackPlugin.js

```diff
@@ -328,13 +333,39 @@ export default class ReactFlightWebpackPlugin {
   resolveAllClientFiles(
     context: string,
     contextResolver: any,
+    normalResolver: any,
     fs: any,
     contextModuleFactory: any,
     callback: (
       err: null | Error,
       result?: $ReadOnlyArray<ClientReferenceDependency>,
     ) => void,
   ) {
+    function hasUseClientDirective(source: string): boolean {
+      if (source.indexOf('use client') === -1) {
+        return false;
+      }
```

Comment:
```suggestion
      if (!source.includes('use client')) {
        return false;
      }
```

Even if it's copypasta, but why not make it more redable at least in this part of the code... WDYT?